### DISCUSSION TITLE
How to show toolbar?

### DISCUSSION THREAD

**Discussion by gegehprast** [community/community#56187] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Copilot `v1.86.82` on `VSCode 1.78.2`

I accidentally clicked Always Show Toolbar and now it's not showing anymore. How do I change this setting? Is this a bug?

![image](https://github.com/community/community/assets/36652820/e5833aca-9178-40e9-b3e4-fc66b36cfed4)

Edit: I should note that this issue only happens on the workspace that I clicked the Always Show Toolbar. The toolbar still shows up on other workspaces, the screenshot I took is from another workspace.

**Comment by davecheney** [comment]
Thank you for raising this. I've forwarded this thread to the development team.

**Comment by hsenag** [comment]
This VSCode issue looks relevant: https://github.com/microsoft/vscode/issues/181764

**Comment by hstaudacher** [comment, marked as answer]
I can verify the issue. Until it's fixed there is a workaround. You can edit your user settings (JSON) and remove the line `"editor.inlineSuggest.showToolbar": "always",` to restore the original behaviour.

**Reply by gegehprast** [reply]
This fixes the issue. Thank you.

**Comment by palcisto** [comment]
I just experienced this very issue today and @hstaudacher's workaround described above worked for returning to the default `onHover` behavior. It would be very beneficial to me if setting to `always` actually always displayed the CoPilot suggestion toolbar when suggestions are being offered.

Here's my version info:
```
Version: 1.78.2
Commit: b3e4e68a0bc097f0ae7907b217c1119af9e03435
Date: 2023-05-10T14:44:45.204Z
Electron: 22.5.2
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin x64 21.3.0
Sandboxed: Yes
```

**Reply by hstaudacher** [reply]
This issues is already [fixed in VSCode](https://github.com/microsoft/vscode/issues/181764). It's available right now using [insiders](https://code.visualstudio.com/insiders/) and will make it to the next VSCode release.
